src/test/java/com/amazon/rdsdata/client/OutputTypesTest.java

```diff
@@ -139,4 +142,8 @@ void shouldThrowExceptionIfFieldCannotBeConvertedToBigInteger() {
                 .isInstanceOf(MappingException.class)
                 .hasMessage(ERROR_CANNOT_CONVERT_TO_TYPE, field, BigInteger.class.toString());
     }
+
+    private enum EnumType {
```

Comment:
not sure about name for this enum type, as well maybe it makes sense to move it to the `util` package as same enum is defined here and in the `InputTypesTest` class - I am open for suggestions

Reply:
Looks OK for me